hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/key/OMKeyRequest.java

```diff
@@ -289,9 +291,11 @@ protected List< OmKeyLocationInfo > allocateBlock(ScmClient scmClient,
     String remoteUser = getRemoteUser().getShortUserName();
     List<AllocatedBlock> allocatedBlocks;
     try {
-      allocatedBlocks = scmClient.getBlockClient()
-          .allocateBlock(scmBlockSize, numBlocks, replicationConfig, serviceID,
-              excludeList, clientMachine);
+      allocatedBlocks = prefetchClient.getBlocks(scmBlockSize, numBlocks, replicationConfig, serviceID, excludeList,
+          clientMachine, clusterMap);
+
+      prefetchClient.prefetchBlocks(scmBlockSize, numBlocks * blockPrefetchFactor, replicationConfig, serviceID,
```

Comment:
Measure the time it takes for `prefetchBlocks` the expectation is that this should be really quick as we fork off a new thread but goog to validate.

Reply:
Mesure the overall `allocateBlock` call as well.